ql/src/java/org/apache/hadoop/hive/ql/parse/RewriteSemanticAnalyzer.java

```diff
@@ -581,6 +581,11 @@ protected ColumnAppender(Table table, HiveConf conf, String subQueryAlias) {
     }
 
     public abstract void appendAcidSelectColumns(StringBuilder stringBuilder, Context.Operation operation);
+   
+    public void appendAcidSelectColumnsForTombstone(StringBuilder stringBuilder, Context.Operation operation) {
```

Comment:
ha, Not a native English speaker and try to understand what's the meaning of `Tombstone `in here? :(
Can we use a more simple word? :)

Reply:
any suggestions?